examples/jsm/loaders/3DMLoader.js

```diff
@@ -509,8 +517,21 @@ Rhino3dmLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 				var width = ctx.measureText( geometry.text ).width + 10;
 				var height = geometry.fontHeight + 10;
 
-				ctx.canvas.width = width;
-				ctx.canvas.height = height;
+				// ref https://bl.ocks.org/duhaime/60c7083009bbd49ce50a58c371d8c818
+				var dpr = window.devicePixelRatio || 1;
+				var bsr = ctx.webkitBackingStorePixelRatio ||
```

Comment:
Hmm, what is this for?

Reply:
See https://discourse.threejs.org/t/simplify-and-improve-code-which-uses-canvastexture/18553

Reply:
Updated that code to only use devicePixelRatio. I would appreciate simplifying all of that while still getting decent quality textures.

Reply:
Thanks!